host-bin/mount-chroot

```diff
@@ -216,32 +228,78 @@ $passphrase" | ecryptfs-wrap-passphrase "$wrappedfnek" -
             tail -c 160 "$KEYFILE" | head -c 80 > "$wrappedkey"
             tail -c 80 "$KEYFILE" > "$wrappedfnek"
 
+            PAD1="`head -c 16 "$wrappedkey"`"
+            PAD2="`tail -c 16 "$wrappedfnek"`"
```

Comment:
this now needs to be `head`...I would look at the rest of the key handling (especially below) to make sure it's up-to-date with the new ...xxx...yyy format.